### PR TITLE
Change endpoint GET and PUT to work ERC + listTypeDefinitionId

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
Related issue : [LPS-175451](https://issues.liferay.com/browse/LPS-175451)

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Review by murilostodolni — APPROVED** [review]
LGTM 🎉 

@marcelabc what do you think? Thank you.

**Comment by marcelabc** [comment]
Just started reviewing :)

File: modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java [hunk 1]
```diff
@@ -68,6 +68,30 @@ public void tearDown() throws Exception {
...
 
+	@Override
+	@Test
+	public void testGetListTypeDefinitionByExternalReferenceCodeListTypeEntriesPageWithSortInteger()
+		throws Exception {
+
+		testGetListTypeDefinitionByExternalReferenceCodeListTypeEntriesPageWithSort(
+			EntityField.Type.INTEGER,
+			(entityField, listTypeEntry1, listTypeEntry2) -> {
+				if (BaseListTypeDefinitionResourceTestCase.BeanTestUtil.
```
Comment (marcelabc) [inline modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java:79]: Hey @pedro-oliveira446,
where did you see this pattern?

Could we use the BeanTestUtil.hasProperty() ? Why do we need to use BaseListTypeDefinitionResourceTestCase.BeanTestUtil?

Comment (pedro-oliveira446) [reply modules/apps/headless/headless-admin-list-type/headless-admin-list-type-test/src/testIntegration/java/com/liferay/headless/admin/list/type/resource/v1_0/test/ListTypeEntryResourceTest.java]: Hi @marcelabc,
I think it was either SF or AFCB that replaced this automatically, but now I have changed it.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by brianchandotcom** [comment]
@pedro-oliveira446 @marcelabc I merged this to master

**Comment by murilostodolni** [comment]
ci:close

**Pull request closed without merging**